stdlib/3/subprocess.pyi

```diff
@@ -43,7 +43,34 @@ if sys.version_info >= (3, 5):
                      stderr: Optional[_TXT] = ...) -> None: ...
         def check_returncode(self) -> None: ...
 
-    if sys.version_info >= (3, 6):
+    if sys.version_info >= (3, 7):
+        # Nearly the same args as for 3.6, except for capture_output and text
+        def run(args: _CMD,
```

Comment:
I think the ordering here isn't quite right. Only the arguments before the `*` in https://github.com/python/cpython/blob/3e26e42c905852394fa136f1cc564dac98b56166/Lib/subprocess.py#L650 should be allowed as positional arguments; everything else should be keyword-only. That includes the two new arguments you're adding, but a couple of the existing arguments (e.g., `input`) should also be keyword-only. Speaking of which, the new `text` argument should also be added to `Popen`.

If you don't want to do all that refactoring, could you just move the two new arguments to be keyword-only?

Reply:
Agh, thanks, fixed.